hub/config/src/main/java/uk/gov/ida/hub/config/data/TransactionConfigRepository.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.ida.hub.config.data;
+
+public class TransactionConfigRepository {
+
+//    private boolean selfServiceEnabled = false; //????
```

Comment:
what's the plan with this? update or clean up and bring in to the new PR?

Reply:
Yes, this should be cleared out for now. It's part of the next step.

Reply:
Removed the class from this branch.